tests/test_helpers.py

```diff
@@ -113,8 +113,16 @@ def test_to_timestamp_none(self):
         # this 'convenience' behaviour has been left left for backwards compatibility
         assert helpers.to_timestamp(None) is None
 
-    def test_from_timestamp(self):
-        assert helpers.from_timestamp(1573431976) == dtm.datetime(2019, 11, 11, 0, 26, 16)
+    def test_from_timestamp_naive(self):
+        datetime = dtm.datetime(2019, 11, 11, 0, 26, 16, tzinfo=dtm.timezone.utc)
+        assert helpers.from_timestamp(1573431976) == datetime
```

Comment:
Here I would test for conversion when `tzinfo=None`, i.e.
```python
datetime = dtm.datetime(2019, 11, 11, 0, 26, 16)
assert helpers.from_timestamp(1573431976, tzinfo=None) == datetime
```
since the other test already takes care of aware datetimes with explicit timezones.